### ISSUE TITLE
Undeclared dependency Syntax::Feature::Junction

### ISSUE THREAD

**Issue by eserte** [mschout/perl-template-plugin-filter-idn#1]
The 0.01 testsuite may fail:
```
#   Failed test 'use Template::Plugin::Filter::IDN;'
#   at t/00-compile.t line 7.
#     Tried to use 'Template::Plugin::Filter::IDN'.
#     Error:  Can't locate Syntax/Feature/Junction.pm in @INC (you may need to install the Syntax::Feature::Junction module) (@INC contains: /home/cpansand/.cpan/build/2018060506/Template-Plugin-Filter-IDN-0.01-1/blib/lib /home/cpansand/.cpan/build/2018060506/Template-Plugin-Filter-IDN-0.01-1/blib/arch /home/cpansand/.cpan/build/2018060506/Net-IDN-Encode-2.400-0/blib/arch /home/cpansand/.cpan/build/2018060506/Net-IDN-Encode-2.400-0/blib/lib /home/cpansand/.cpan/build/2018060506/Net-IDN-Encode-2.400-0/blib/arch /home/cpansand/.cpan/build/2018060506/Net-IDN-Encode-2.400-0/blib/lib /usr/perl5.18.4p/lib/site_perl/5.18.4/amd64-freebsd /usr/perl5.18.4p/lib/site_perl/5.18.4 /usr/perl5.18.4p/lib/5.18.4/amd64-freebsd /usr/perl5.18.4p/lib/5.18.4 .) at /usr/perl5.18.4p/lib/site_perl/5.18.4/syntax.pm line 111.
# BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2018060506/Template-Plugin-Filter-IDN-0.01-1/blib/lib/Template/Plugin/Filter/IDN.pm line 7.
# Compilation failed in require at t/00-compile.t line 7.
# BEGIN failed--compilation aborted at t/00-compile.t line 7.
# Looks like you failed 1 test of 1.
t/00-compile.t ........... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by eserte** [comment]
The 0.02 testsuite, too, may fail.

**Comment by mschout** [comment]
Thanks! Fixed for v0.03 (going to cpan shortly)
